src/Libraries/RevitNodes/Elements/InternalUtilities/ElementUtils.cs

```diff
@@ -48,5 +51,65 @@ public static void UpdateLevelName(ref string name)
                 name += "(1)";
             }
         }
+
+
+        /// <summary>
+        /// Get a revit parameters value
+        /// </summary>
+        /// <param name="param">Revit parameter</param>
+        /// <returns></returns>
+        public static object GetParameterValue(Autodesk.Revit.DB.Parameter param)
```

Comment:
this is going to be visible in the library and is going to expose the `Autodesk.Revit.DB` namespace in Dynamo, either this should be internal or suppressed - we should not expose this namespace in Dynamo.